modules/efk-logging-fluentd-throttling.adoc

```diff
@@ -42,7 +42,7 @@ $ oc edit configmap/fluentd
 The format of the *_throttle-config.yaml_* key is a YAML file that contains
 project names and the desired rate at which logs are read in on each
 node. The default is 1000 lines at a time per node. For example:
-
++
 ----
 throttle-config.yaml: |
   - opensift-logging:
```

Comment:
@richm Where in the config map is this code added? I couldn't get the code to be accepted.

Reply:
Throttling is not supported (yet) by rsyslog.